packages/loader/container-loader/src/loader.ts

```diff
@@ -212,6 +212,15 @@ export class Loader extends EventEmitter implements ILoader {
         });
     }
 
+    public cacheContainer(container: Container, request: IRequest, parsedUrl: IParsedUrl) {
+        const { canCache } = this.parseHeader(parsedUrl, request);
+
+        if (canCache) {
+            const versionedId = this.getKeyForContainerCache(request, parsedUrl);
```

Comment:
Nit: Even though it may be a versionID, the API returns a key, so I'd name variables same way